src/meta_schedule/task_scheduler/task_scheduler.cc

```diff
@@ -124,7 +124,7 @@ void TaskSchedulerNode::Tune() {
 
   int running_tasks = tasks.size();
   for (int task_id; (task_id = NextTaskId()) != -1;) {
-    LOG(INFO) << "Scheduler picks Task #" << task_id << ": " << tasks[task_id]->task_name;
+    LOG(INFO) << "Scheduler picks Task #" << task_id + 1 << ": " << tasks[task_id]->task_name;
```

Comment:
Hi Xiyou! I'm curious about this change. After this change the printed ids at several places differ (here with `+ 1` while others without.)

https://github.com/apache/tvm/blob/0c836b73ffd9669bcc416515dce6436cbd7d7ebe/src/meta_schedule/task_scheduler/task_scheduler.cc#L96

https://github.com/apache/tvm/blob/0c836b73ffd9669bcc416515dce6436cbd7d7ebe/src/meta_schedule/measure_callback/echo_statistics.cc#L228

Perhaps it's better to keep them consistent :eyes:?

Reply:
Hi, thanks for pointing that out. The reason for this change is previously we have `task 0/18` to `task 17/18` which seems confusing to me. I intend to make sure the log is consistent but I might missed a couple places. I'll make a thorough check to update that.